<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite hostname="Alkesh-PC" name="Test" tests="2" failures="2" timestamp="8 Sep 2019 06:06:49 GMT" time="89.096" errors="0">
  <testcase name="facebook" time="6.198" classname="com.qa.maven.FirstTest">
    <failure type="java.lang.AssertionError" message="expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: expected [true] but found [false]
	at com.qa.maven.FirstTest.facebook(FirstTest.java:40)
	at java.util.concurrent.FutureTask.run(FutureTask.java:266)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1142)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:617)
	at java.lang.Thread.run(Thread.java:745)
... Removed 20 stack frames]]>
    </failure>
  </testcase> <!-- facebook -->
  <testcase name="website" time="10.279" classname="com.qa.maven.SecondTest">
    <failure type="java.lang.AssertionError" message="expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: expected [true] but found [false]
	at com.qa.maven.SecondTest.website(SecondTest.java:19)
	at java.util.concurrent.FutureTask.run(FutureTask.java:266)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1142)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:617)
	at java.lang.Thread.run(Thread.java:745)
... Removed 20 stack frames]]>
    </failure>
  </testcase> <!-- website -->
</testsuite> <!-- Test -->
